frontstage/views/account/account_survey_share.py

```diff
@@ -129,20 +129,22 @@ def send_instruction_get(session):
                            business_name=selected_business[0]['name'])
 
 
-def build_payload():
+def build_payload(respondent_id):
     """
         This method builds payload required for the party endpoint to register new pending shares.
         TODO: The logic should change for multiple business once the story is in play.
         payload example:
         {  pending_shares: [{
             "business_id": "business_id"
             "survey_id": "survey_id",
-            "email_address": "email_address"
+            "email_address": "email_address",
+            "shared_by": "respondent_id"
```

Comment:
You sure you want the give the string "respondent_id" instead of the id? ^^  Need to remove the quotes

Reply:
will change it